lib/symengine.rb

```diff
@@ -33,6 +33,19 @@ def Function(n)
     def evalf(operand, prec=53, real=false)
         return _evalf(operand, prec, real)
     end
+    def lambdify(exp)
+      sym_map = exp.free_symbols.map { |sym| sym.to_s}.join(",")
+      str = exp.to_s
+      str.gsub!(/[\d\.]+/, 'Rational(\0,1)')
+      replacements = [ 
+                   ["asin", "Math.asin"], ["acos", "Math.acos"], ["atan","Math.atan"],
+                   ["sin", "Math.sin"], ["cos", "Math.cos"], ["tan","Math.tan"],
+                   ["asinh", "Math.asinh"], ["acosh", "Math.acosh"], ["atanh","Math.atanh"],
+                   ["sinh", "Math.sinh"], ["cosh", "Math.cosh"], ["tanh","Math.tanh"],
+                   ["pi", "Math::PI"], ["E", "Math::E"] ]
+      replacements.each {|replacement| str.gsub!(/(\b#{replacement[0]}\b)/, replacement[1])}
+      return eval("lambda { |"+ sym_map +"| "+ str +" }")
```

Comment:
String interpolation is preferred method of creating complex strings (`"lambda { |#{sym_map}| #{str} }"`). And `return` keyword is unnecessary here :)